plugins/org.locationtech.udig.catalog/src/org/locationtech/udig/catalog/IProcess.java

```diff
@@ -101,10 +97,11 @@
         if (adaptee.isAssignableFrom(IServiceInfo.class)) {
             try {
                 monitor.beginTask("service info", 100); //$NON-NLS-1$
-                IService service = service(SubMonitor.convert(monitor, 40));
-                IServiceInfo info = service.createInfo(SubMonitor.convert(monitor, 60));
-                return adaptee.cast(info);
-            } finally {
+                IService service = service( new SubProgressMonitor(monitor,40));
+                IServiceInfo info = service.createInfo( new SubProgressMonitor(monitor,60) );
+                return adaptee.cast( info );
+            }
+            finally {
```

Comment:
Hmm seems odd, I guess something went wrong (rebase??) since here you use the old SubProgressMonitor instead of SubMonitor again

Reply:
I reverted one commit so this file is not changed within this PR at all.